### DISCUSSION TITLE
Elastic agent install

### DISCUSSION THREAD

**Discussion by Heheheuebd** [Security-Onion-Solutions/securityonion#13584] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

1tb

### Storage for /nsm

500gb

### Network Traffic Collection

span port

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello,

My company has strict policies that do not allow unsigned exes. We would like to install the elastic agent on windows computers. Is there any documentation on configuring the elastic agent that is signed by elastic to work with so? I’d imagine we would need to import the self signed cert and join to the fleet. Is there any documentation on this?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by bushcraftbuddy** [comment]
This should get you going in the right direction

https://www.elastic.co/guide/en/fleet/8.10/secure-connections.html

Also, this might have some workarounds

https://www.elastic.co/guide/en/fleet/8.10/install-fleet-managed-elastic-agent.html#elastic-agent-installation-steps

**Comment by reyesj2** [comment]
The link above https://www.elastic.co/guide/en/fleet/8.10/install-fleet-managed-elastic-agent.html#elastic-agent-installation-steps should get you going, you will need to have the CA cert from the Security Onion manager ```/etc/pki/ca.crt``` as a trusted cert on the windows endpoints for the connection to be established securely from endpoints <-> SO-manager.

Ensure you've allowed the endpoints ip addresses or network through the Security Onion firewall as well https://docs.securityonion.net/en/2.4/elastic-agent.html#deployment
